### PR TITLE
Edited Blinky to run with latest Gpio API

### PR DESCRIPTION
**Pull request by maiorfi**
## Description
Gpio port bound to led should be explicitly set as Output mode pin

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/nanoframework/Samples?pullRequest=40) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/nanoframework/Samples?pullRequest=40) before we can accept your contribution.<br/><br/>:x: maiorfi [sign now](https://cla.dotnetfoundation.org/nanoframework/Samples?pullRequest=40)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/nanoframework/Samples?pullRequest=40) it.</sub>

**Comment by networkfusion** [comment]
Umm, the latest api is 'System.Device'. 'Windows.Devices' is the old one...

File: samples/Blinky/Blinky/Blinky.nfproj [hunk 1]
```diff
@@ -22,11 +22,21 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="mscorlib, Version=1.10.1.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
```
Comment (josesimoes) [inline samples/Blinky/Blinky/Blinky.nfproj:25]: This adding a second reference to mscorlib. Can't happen.

File: samples/Blinky/Blinky/Blinky.nfproj [hunk 2]
```diff
@@ -22,11 +22,21 @@
...
+      <HintPath>..\packages\nanoFramework.CoreLibrary.1.10.1-preview.9\lib\mscorlib.dll</HintPath>
+      <Private>True</Private>
+      <SpecificVersion>True</SpecificVersion>
+    </Reference>
     <Reference Include="mscorlib, Version=1.9.1.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
       <HintPath>..\packages\nanoFramework.CoreLibrary.1.9.1-preview.6\lib\mscorlib.dll</HintPath>
       <Private>True</Private>
       <SpecificVersion>True</SpecificVersion>
     </Reference>
+    <Reference Include="nanoFramework.Runtime.Events, Version=1.9.0.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
```
Comment (josesimoes) [inline samples/Blinky/Blinky/Blinky.nfproj:35]: This adding a second reference to Events library. Can't happen.

File: samples/Blinky/Blinky/Program.cs [hunk 3]
```diff
@@ -1,11 +1,6 @@
-﻿//
```
Comment (josesimoes) [inline samples/Blinky/Blinky/Program.cs:1]: The copyright/licensing header is required. Can't remove it.

**Review by josesimoes — CHANGES_REQUESTED** [review]
Hi Lorenzo. Glad to see you here! 🙂
Seems that you've reverted this to use the deprecated Windows.Devices.Gpio...
The sample was updated to use the new System.Device.Gpio API. It should reflect the move towards the new API going forward. It's OK to update the refs to the latest NuGet versions but we shouldn't move backwards.

**Comment by maiorfi** [comment]
Ok, I will re-run each step I followed to solve what I believed being an error due to blinky sample not updated to latest nuget package.

**Comment by josesimoes** [comment]
@maiorfi I can see what you're seeing: there were several of the commented calls that were still on the older API format of OpenPin.
Sample just updated and the calls updated too.

Let me know if this works for you now. 😉

**Pull request closed without merging**